understanding/21/text-spacing.html

```diff
@@ -23,7 +23,8 @@ <h2>Intent</h2>
          <p>This SC focuses on the adaptability of content to an increase in spacing between lines, words, letters, and paragraphs. Any combination of these may assist a user with effectively reading text. As well, ensuring that content correctly adapts when users override author settings for spacing also significantly increases the likelihood other style preferences can be set by the user. For example, a user may need to change to a wider font family than the author has set in order to effectively read text. </p>
       	<section>
          <h3 id="authorresp">Author Responsibility </h3>
-         <p>This SC does not dictate that authors must set all their content to the specified metrics. Rather, it specifies that an author's content has the ability to be set to those metrics without loss of content or functionality. The author requirement is both to not interfere with a user's ability to override the author settings, and to ensure that content thus modified does not break content in the manners shown in figures 1 through 3 in Effects of Not Allowing for Spacing Override. The values in the SC are a baseline. Authors are encouraged to surpass the values specified, not see them as a ceiling to build to. If the user chooses to go beyond the metrics specified any resulting loss of content or functionality is the user's responsibility.</p>
+         <p>This SC does not dictate that authors must set all their content to the specified metrics, nor does the SC intend to imply that all users will adjust the specified metrics. Rather, it specifies that an author's content has the ability to be set to those metrics without loss of content or functionality. The author requirement is both to not interfere with a user's ability to override the author settings, and to ensure that content thus modified does not break content in the manners shown in figures 1 through 3 in Effects of Not Allowing for Spacing Override. The values in the SC are a baseline. For content written in languages where the specified metrics would help increase readability, authors are encouraged to surpass the values specified, not see them as a ceiling to build to. These metrics are not meant to represent a ceiling for authors to build to, nor as minimum requirements to adhere to, as some metrics may not even be applicable to certain languages. For example, languages such as Japanese do not use spacing following paragraphs, meaning that users won't necessarily force any paragraph spacing metrics in practice. If the user chooses to go beyond the metrics specified any resulting loss of content or functionality is the user's responsibility.</p>
```

Comment:
I have heard that China has some related typographic guidelines, and suspect that that may also be the case for Japanese.  Therefore, i'd suggest adding a sentence or paragraph to recommend that content should be amenable to supporting any local guidelines for adjusting typography in order to improve readability.

Reply:
How about "It is benficial for users if authors use any locally available guidance for improving readability in the local language or writing system."

NB: We have to avoid RFC-like words such as "should" for this sort of recommendation, some people see that as adding requirements by the back door.

Reply:
It appears that neither the preview button nor the diff button work.

Reply:
@murata2makoto, no they are auto-generated but don't account for how the WCAG spec works. I can make a manual [preview of the understanding document](https://raw.githack.com/w3c/wcag/Issue3338-text-spacing-i18n/understanding/21/text-spacing.html), but not the SC text.

The whole [SC text file](https://github.com/w3c/wcag/pull/3347/files#diff-1db1537d239fb0076187e01dc53b9d8c2a0dedb423e89350a445dbb922c640f8) is only 20 lines, so should be ok to review as code.

Reply:
The suggested line has been added.